addons/l10n_pe/data/res_country_data.xml

```diff
@@ -9,22 +9,16 @@
                 <div class="o_address_format">
                     <field name="parent_id" invisible="1"/>
                     <field name="type" invisible="1"/>
-                    <field name="street_name" placeholder="Street Name..." attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}" class="oe_read_only"/>
-                    <field name="street" placeholder="Street" class="oe_edit_only"/>
-                    <field name="street2" placeholder="Street2" invisible="1"/>
-                    <div class="o_row">
-                        <label for="street_number" class="oe_edit_only"/>
-                        <field name="street_number" attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}"/>
-                        <span> </span>
-                        <label for="street_number2" class="oe_edit_only"/>
-                        <field name="street_number2" attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}"/>
-                    </div>
-                    <field name="l10n_pe_district" placeholder="District..."/>
-                    <field name="city" placeholder="Province..." invisible="1"/>
-                    <field name="city_id" placeholder="Province..."/>
-                    <field name="state_id" class="o_address_state" placeholder="State..." options='{"no_open": True}'/>
-                    <field name="country_id" placeholder="Country" class="o_address_country" options='{"no_open": True, "no_create": True}'/>
-                    <field name="zip" placeholder="ZIP" class="o_address_zip"/>
```

Comment:
Because it is using the mechanism of the address_view_id and we needed more adaptations before which were not necessarily practical in the companies of a different country. For the stable policy, I think it is better to keep it, although with an attrs we could get rid of it.

Reply:
So if I understand correclty, you are just dropping the use of `base_address_extended` (and `base_address_city` is doing it's thing automatically like intended)?

However, the correct values should be set when you input the `country_id`, aren't they?

Reply:
Indeed.  What happens now, is that if you put a district, the country will change to PE and usability wise it does not seem bad this way.  For the address view, it is the country of the company that counts.

Reply:
@william-andre It could be done like that in master (I need to check further), but I don't want to add it in the views in stable.

Reply:
What's the difference with this (regarding the stable policy)?

Reply:
@william-andre That it is only applied on companies with countries which have the address_view_id applied. Maybe someone already removed the address_view_id and did something himself.  This is far less intrusive than changing it for all the partners.